<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>759</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="vl_main">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Infrared Camera Controls</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="hl_irimager_controls">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>IR Serial</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_irimager_usb_sn">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>XXXXXXXXX</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="b_toggle_usb_conn">
             <property name="text">
              <string>Connect via USB</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>HW Temp Range:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbox_hw_temp_range"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>IR Camera Frame Metadata:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Counter</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>CounterHW</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>FlagState</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>TempBox</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>TempFlag</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Timestamp</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="spacing">
            <number>7</number>
           </property>
           <item>
            <widget class="QLabel" name="l_counter">
             <property name="text">
              <string>Counter</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="l_counterHW">
             <property name="text">
              <string>CounterHW</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="l_flagstate">
             <property name="text">
              <string>FlagState</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="l_tempbox">
             <property name="text">
              <string>TempBox</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="l_tempflag">
             <property name="text">
              <string>TempFlag</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="l_timestamp">
             <property name="text">
              <string>Timestamp</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="minimumSize">
              <size>
               <width>415</width>
               <height>325</height>
              </size>
             </property>
             <property name="title">
              <string>IR View:</string>
             </property>
             <widget class="PlotWidget" name="gfx_thermal">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>20</y>
                <width>400</width>
                <height>300</height>
               </rect>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>400</width>
                <height>300</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>400</width>
                <height>300</height>
               </size>
              </property>
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
              <property name="sizeAdjustPolicy">
               <enum>QAbstractScrollArea::AdjustToContents</enum>
              </property>
              <property name="sceneRect">
               <rectf>
                <x>0.000000000000000</x>
                <y>0.000000000000000</y>
                <width>0.000000000000000</width>
                <height>0.000000000000000</height>
               </rectf>
              </property>
              <property name="renderHints">
               <set>QPainter::LosslessImageRendering</set>
              </property>
             </widget>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="minimumSize">
              <size>
               <width>415</width>
               <height>325</height>
              </size>
             </property>
             <property name="title">
              <string>Mono View:</string>
             </property>
             <widget class="PlotWidget" name="gfx_monoc">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>20</y>
                <width>400</width>
                <height>300</height>
               </rect>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>400</width>
                <height>300</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>400</width>
                <height>300</height>
               </size>
              </property>
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
              <property name="sizeAdjustPolicy">
               <enum>QAbstractScrollArea::AdjustToContents</enum>
              </property>
              <property name="sceneRect">
               <rectf>
                <x>0.000000000000000</x>
                <y>0.000000000000000</y>
                <width>0.000000000000000</width>
                <height>0.000000000000000</height>
               </rectf>
              </property>
              <property name="renderHints">
               <set>QPainter::LosslessImageRendering</set>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Monochrome Camera Controls</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QPushButton" name="b_mono_toggle">
              <property name="text">
               <string>Autodetect</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>932</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
